deepmd_utils/model_format/network.py

```diff
@@ -156,10 +150,13 @@ def __init__(
         idt: Optional[np.ndarray] = None,
         activation_function: Optional[str] = None,
         resnet: bool = False,
+        precision: str = "default",
```

Comment:
Why not just set `float64` as the default.. In deepmd tf, `default` may have different meanings, but I think we need the exact meaning when storing the model.